netbox/dcim/signals.py

```diff
@@ -210,7 +210,7 @@ def sync_cached_scope_fields(instance, created, **kwargs):
     for model in (Prefix, Cluster, WirelessLAN):
         qs = model.objects.filter(**filters)
 
-        for obj in qs.only('id'):
+        for obj in qs:
```

Comment:
I contemplated doing something like the following, but it felt like the wrong solution, fragile and a pain to maintain.

```
fields = ['id'] if model != Prefix else ['id', 'prefix', 'vrf_id']
for obj in qs.only(*fields):
    ...
```